dockerfiles/business-process/Dockerfile

```diff
@@ -16,8 +16,76 @@
 #
 # ------------------------------------------------------------------------
 
-# set to product base image
-FROM wso2ei-base:6.1.1
+# set to latest Ubuntu LTS
+FROM ubuntu:16.04
+MAINTAINER WSO2 Docker Maintainers "dev@wso2.org"
+
+# set user configurations
+ARG USER=wso2carbon
+ARG USER_ID=802
+ARG USER_GROUP=wso2
+ARG USER_GROUP_ID=802
+ARG USER_HOME=/home/${USER}
+# build arguments for WSO2 product installation
+ARG WSO2_SERVER_NAME=wso2ei
+ARG WSO2_SERVER_VERSION=6.1.1
+ARG WSO2_SERVER_REPOSITORY=product-ei
```

Comment:
@RidmiR this build argument is not required, AFAIU. This was originally added to the latest Docker resources to provide the GitHub repository path. If this has been added, IMO we should remove it.

Please correct me if I have missed anything. And it applies to all files.

Reply:
@chirangaalwis yes we dont need this. Corrected in the latest commit.